content/concept-amendments.md

```diff
@@ -114,22 +114,22 @@ The following is a comprehensive list of all known amendments and their status o
 | Name                                  | Introduced | Status                  |
 |:--------------------------------------|:-----------|:------------------------|
 | [FlowCross](#flowcross)               | v0.70.0    | [Planned: TBD]( "BADGE_LIGHTGREY") |
-| [EnforceInvariants](#enforceinvariants) | v0.70.0  | [Expected: 2017-06-29]( "BADGE_BLUE") |
-| [fix1373](#fix1373)                   | v0.70.0    | [Expected: 2017-06-29]( "BADGE_BLUE") |
 | [SHAMapV2](#shamapv2)                 | v0.33.0    | [Planned: TBD]( "BADGE_LIGHTGREY") |
 | [OwnerPaysFee](#ownerpaysfee)         | v0.33.0    | [Planned: TBD]( "BADGE_LIGHTGREY") |
 | [Tickets](#tickets)                   | N/A        | [Planned: TBD]( "BADGE_LIGHTGREY") |
-| [fix1368](#fix1368)                   | v0.60.0    | [Enabled: 2017-03-31](https://charts.ripple.com/#/transactions/3D20DE5CD19D5966865A7D0405FAC7902A6F623659667D6CB872DF7A94B6EF3F "BADGE_GREEN") |
-| [PayChan](#paychan)                   | v0.33.0    | [Enabled: 2017-03-31](https://charts.ripple.com/#/transactions/16135C0B4AB2419B89D4FB4569B8C37FF76B9EF9CE0DD99CCACB5734445AFD7E "BADGE_GREEN") |
-| [Escrow](#escrow)                     | v0.60.0    | [Enabled: 2017-03-31](https://charts.ripple.com/#/transactions/C581E0A3F3832FFFEEB13C497658F475566BD7695B0BBA531A774E6739801515 "BADGE_GREEN") |
-| [TickSize](#ticksize)                 | v0.50.0    | [Enabled: 2017-02-21](https://charts.ripple.com/#/transactions/A12430E470BE5C846759EAE3C442FF03374D5D73ECE5815CF4906894B769565E "BADGE_GREEN") |
+| [EnforceInvariants](#enforceinvariants) | v0.70.0  | [Enabled: 2017-07-07](https://xrpcharts.ripple.com/#/transactions/7EBA3852D111EA19D03469F6870FAAEBF84C64F1B9BAC13B041DDD26E28CA399 "BADGE_GREEN") |
```

Comment:
tx hash should be `17593B03F7D3283966F3C0ACAF4984F26E9D884C9A202097DAED0523908E76C6`